vcd/sample_vcd_test_config.json

```diff
@@ -29,7 +29,10 @@
     "//": "Catalog name to be created (if using testEnvBuild) or to reuse if exists already",
     "catalog": {
       "name": "tf_catalog",
-      "catalogItem": "tf_catalog_item"
+      "catalogItem": "tf_catalog_item",
+      "catalogItemWithMultiVms": "item_with_multi_vms",
+      "vmName1": "first_vm_name_in_multi_vm_item",
+      "vmName2": "second-vm_name_in_multi_vm_item"
```

Comment:
Missing `ovaMultiVmPath` in this example

Reply:
wrong file. Compared it with `vendor` dir

Reply:
It happened to me as well. Perhaps we should purge the sample files from vendor?